src/components/Popups/__tests__/PopupReply-test.tsx

```diff
@@ -52,5 +52,19 @@ describe('PopupReply', () => {
             expect(wrapper.exists(PopupBase)).toBe(true);
             expect(wrapper.exists(ReplyForm)).toBe(true);
         });
+
+        test('should maintain referential integrity of the PopupBase options object across renders', () => {
+            const wrapper = getWrapper();
+
+            const popupOptions = wrapper.find(PopupBase).prop('options');
```

Comment:
Is a new test needed for testing IE/non-IE?

Reply:
added, thanks